packages/react-native/src/Authenticator/Defaults/ConfirmVerifyUser/ConfirmVerifyUser.tsx

```diff
@@ -1,17 +1,42 @@
 import React from 'react';
-import { Text } from 'react-native';
 
 import { DefaultFooter } from '../../common/DefaultFooter';
-import { DefaultFormFields } from '../../common/DefaultFormFields';
 import { DefaultHeader } from '../../common/DefaultHeader';
+import { Button, ErrorMessage } from '../../../primitives';
 import { DefaultConfirmVerifyUserComponent } from '../types';
+import { authenticatorTextUtil } from '@aws-amplify/ui';
+import { styles } from './styles';
 
-const ConfirmVerifyUser: DefaultConfirmVerifyUserComponent = () => {
-  return <Text>ConfirmVerifyUser</Text>;
+const { getAccountRecoveryInfoText, getSkipText } = authenticatorTextUtil;
+
+const ConfirmVerifyUser: DefaultConfirmVerifyUserComponent = ({
+  error,
+  fields,
+  FormFields,
+  Footer,
+  Header,
+  isPending,
+}) => {
+  return (
+    <>
+      <Header>{getAccountRecoveryInfoText()}</Header>
+      <FormFields isPending={isPending} fields={fields} />
+      {error ? <ErrorMessage>{error}</ErrorMessage> : null}
+      <Button
+        style={styles.buttonSecondary}
+        textStyle={styles.buttonSecondaryLabel}
+      >
+        {getSkipText()}
+      </Button>
+      <Footer />
```

Comment:
Just curious if there's no secondary button for this screen?

Reply:
Didn't look like it to me, double checked with the react web version as well.